test/helpers/kubeclient_helper.rb

```diff
@@ -0,0 +1,21 @@
+module KubeclientHelper
+  def kubeclient
+    @kubeclient ||= begin
+      config = Kubeclient::Config.read(ENV["KUBECONFIG"])
+      unless config.contexts.include?("minikube")
+        raise "`minikube` context must be configured in your KUBECONFIG (#{ENV["KUBECONFIG"]})"
```

Comment:
This was just logging before. Was there a reason not to raise? Seems to me the tests calling this method shouldn't try to run without it.